docusaurus/docusaurus.config.js

```diff
@@ -48,6 +51,32 @@ const defaultPlugins = SDK_FOLDERS.map((SDK) => {
             sidebarPath: require.resolve(sidebarPath),
           }
         : {}),
+      admonitions: {
+        infima: true,
+        customTypes: {
+          note: {
+            ifmClass: 'note',
+            svg: '<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M7 13.7A6.7 6.7 0 117 .3a6.7 6.7 0 010 13.4zm0-1.4A5.3 5.3 0 107 1.7a5.3 5.3 0 100 10.6zm-.7-8.6h1.4V5H6.3V3.7zm0 2.6h1.4v4H6.3v-4z" fill="#000" fill-opacity=".4"/></svg>'
```

Comment:
Can we move svg content to separate files somehow? And import then in docusaurus.config.js

Reply:
> Can we move svg content to separate files somehow? And import then in docusaurus.config.js

@vishalnarkhede @tsirlucas The admonition component only accepts inline svgs strings (no file reference like 'img/icon.svg') so I decided to move those into another file and reference them _as strings_ in the config, let me know what you think of this solution